### DISCUSSION TITLE
Refresh source physical file within an object filter

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1291] [IBM i]
Would it be possible to have a refresh on each item within an object filter?  The refresh is nice on the object browser, but often times I don't care about refreshing the entire object filter, but just one (in the below case) QRPGLESRC.
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/132368707/0678a00f-05c9-438f-a81d-b983f3d21535)

**Comment by sebjulliand** [comment]
@worksofliam, I can have a look at that.
